core/evancore.vue.libs/src/components/table/table.vue

```diff
@@ -68,28 +86,83 @@ export default Component;
     margin-left: auto;
     margin-right: auto;
 
+    // This can create janky behavior with sticky table header
+    // Without this we have trouble with borders on rows
+    border-collapse: separate;
+    border-spacing: 0 5px;
+
     & > thead.evan-table-head > tr.evan-table-head-row > th {
       background-color: cssVar('bg-level-3');
       border: none;
       color: cssVar('gray-600');
+      font-size: 10px;
+      // Using values from figma creates weird shadow bleeding
+      box-shadow: 2px 3px 3px rgba(0, 0, 0, 0.15);
+
+      &[aria-sort] {
+        // remove standard arrows
+        background-image: none;
+      }
 
       &[aria-sort='ascending'],
       &[aria-sort='descending'] {
         color: cssVar('primary');
+
+        &::after {
+          // 14px to avoid janky height changes on sort
+          height: 14px;
+          width: 14px;
+          vertical-align: sub;
+          display: inline-block;
+          margin-left: 6px;
+        }
+      }
+      &[aria-sort='ascending']::after {
+        content: url('data:image/svg+xml;utf-8,' + $arrow-up);
+      }
+      &[aria-sort='descending']::after {
+        content: url('data:image/svg+xml;utf-8,' + $arrow-down);
+      }
+
+      &.th-important {
+        width: 200px;
+      }
+      &.th-date {
+        width: 120px;
+      }
+      &.th-icon {
+        width: 70px;
       }
     }
 
     & > tbody > tr.evan-table-body-row {
-      height: 64px;
+      height: 56px;
       background-color: white;
+      color: cssVar('gray-600');
       cursor: pointer;
+
       // create spacing between rows without using border-collapse
       // because it causes janky behavior with sticky header
-      border-bottom: 4px solid cssVar('bg-level-3');
+      // border-bottom: 4px solid cssVar('bg-level-3');
+
+      &:hover {
+        background-color: cssVar('gray-100');
+        // border creates janky jumping. Using inset shadow as an alternative
+        box-shadow: inset 0px 0px 0px 1px cssVar('gray-200');
```

Comment:
you may avoid the jumpin adding a transparent border in general and just set the color on hover.

Reply:
Tried it before, but didn't work. Could spend more time with it, but inset shadows work for now and are supported in all browsers.